descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,22 @@
   "id": "@project.artifactId@-@project.version@",
   "name": "Camnuda BPM Module",
   "provides": [
+    {
+      "id": "camunda-history",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/camunda/history/process-instance",
+          "permissionsRequired": ["camunda.history.process-instance.collection.get"]
```

Comment:
These permissions are created without consideration to ongoing plans:
- https://folio-org.atlassian.net/browse/MODCAMUNDA-4
- https://folio-org.atlassian.net/browse/MODWRKFLOW-15

Reply:
Not true.